android/src/main/java/com/swmansion/rnscreens/Screen.java

```diff
@@ -151,7 +152,22 @@ public void setTransitioning(boolean transitioning) {
       return;
     }
     mTransitioning = transitioning;
-    super.setLayerType(transitioning ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_NONE, null);
+    boolean isWebViewInScreen = findWebView(this);
+    super.setLayerType(transitioning && !isWebViewInScreen ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_NONE, null);
+  }
+
+  private boolean findWebView(ViewGroup viewGroup) {
+    for(int i = 0; i < viewGroup.getChildCount(); i++) {
+      View child = viewGroup.getChildAt(i);
+      if (child instanceof WebView) {
+        return true;
+      } else if (child instanceof ViewGroup) {
+         if (findWebView((ViewGroup) child)) {
```

Comment:
`find` should probably give back the `WebView` itself, not a `boolean` value

```suggestion
    boolean isWebViewInScreen = hasWebView(this);
    super.setLayerType(transitioning && !isWebViewInScreen ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_NONE, null);
  }

  private boolean hasWebView(ViewGroup viewGroup) {
    for(int i = 0; i < viewGroup.getChildCount(); i++) {
      View child = viewGroup.getChildAt(i);
      if (child instanceof WebView) {
        return true;
      } else if (child instanceof ViewGroup) {
         if (hasWebView((ViewGroup) child)) {
```